### ISSUE TITLE
Add gh activity indicator

### ISSUE THREAD

**Issue by mradkov** [hackbg/hack-bg-website#70]
Add some element that will indicate the activity we have on gh.

- we can fetch the data using the github api
- think of a way to present it (pulse, graph, blinking dot or sth else).

**Comment by AtanasKrondev** [comment]
What type of activity do you mean?
Like commits on any repository on https://github.com/hackbg or creating new repository?
Should it be like mini-feed or like 'N commits for the last week' in the footer maybe?

**Comment by mradkov** [comment]
Yes, something like this, but I think its better to discuss it first.

I thought any github contribution by our team and not only in our organization, but the GitHub api is not so generous so we'll have to do it ourselves
